anvil-csharp-core.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Would Unity remove this file, or just ignore it? I assume it's harmless

Reply:
Yea, seems to be ignored. I haven't seen any weird side effects...yet.